envi/archs/thumb16/disasm.py

```diff
@@ -109,14 +109,14 @@ def rd_pc_imm8(va, value):  # add
 
 def rt_pc_imm8(va, value): # ldr
     rt = shmaskval(value, 8, 0x7)
-    imm = e_bits.signed(shmaskval(value, 0, 0xff), 1) * 4
```

Comment:
yup.  that's why i did this :)  thought of you when i made the change, actually.